### PR TITLE
METAL-739: Adding update risk to 4.11.51

### PR DESCRIPTION
**Pull request by LalatenduMohanty**

### CODE REVIEW

**Review by wking — APPROVED** [review]
/lgtm

**Comment by wking** [comment]
/retitle METAL-739: Adding update risk to 4.11.51

File: blocked-edges/4.11.51-incomptiable-python-update-breaking-baremetal-provisioning.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+to: 4.11.51
+from: .*
+url: https://issues.redhat.com/browse/METAL-739
+name: BrokenBaremetalProvisioning
+message: The target release fails to provision and de-provision new baremetal machines required for scaling of a cluster.
+matchingRules:
+- type: PromQL
+  promql:
+    promql: |
+      group(cluster_infrastructure_provider{type=~"BareMetal|None"})
```
Comment (sdodson) [inline blocked-edges/4.11.51-incomptiable-python-update-breaking-baremetal-provisioning.yaml:10]: This should only be Bare metal, the affected component is only used there.

**Comment by wking** [comment]
[e2e][1]:

```
Verifying all releases are uploaded
Error: Missing the following versions in scraped images: {Version { major: 4, minor: 11, patch: 51, pre: [], build: [] }}
```

So this will sit until there's actually a 4.11.51 release that exists (and makes it into the `candidate` channel?), and then we should be able to merge after a retest.

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cincinnati-graph-data/4227/pull-ci-openshift-cincinnati-graph-data-master-e2e/1712599414930935808#1:build-log.txt%3A22-23

**Review by sdodson — CHANGES_REQUESTED** [review]

File: blocked-edges/4.11.51-incomptiable-python-update-breaking-baremetal-provisioning.yaml (hunk 1 above)
Comment (LalatenduMohanty) [reply blocked-edges/4.11.51-incomptiable-python-update-breaking-baremetal-provisioning.yaml]: Fixed it.

**Review by wking — APPROVED** [review]
/lgtm
/hold cancel

**Pull request merged**